client/rechallenge/throwers.js

```diff
@@ -49,13 +49,13 @@ const throwForJsHtml = {
         }
       }
     }, {
-      name: 'gomix in code',
-      description: 'Code with the URL gomix.me ' +
+      name: 'glitch in code',
+      description: 'Code with the URL glitch.me ' +
```

Comment:
@ZZhaoTireless Could you change these to check for either `glitch.com` or `glitch.me`? I think the former is much more likely but the latter still redirects so it might as well be both.